Model/Quote/Checkout.php

```diff
@@ -262,15 +262,17 @@ private function getDeliveryOptionsStrings()
     }
 
     /**
-     * @param $carrier
+     * @param array       $carrier
+     * @param string|null $country
      *
      * @return string
      */
-    public function checkPackageType(array $carrier): string
+    public function checkPackageType(array $carrier, string $country = null): string
```

Comment:
`?string`, nu mag je er niet zelf `null` in stoppen

Reply:
done